galley/pkg/config/source/kube/apiserver/source_dynamic_test.go

```diff
@@ -367,7 +368,10 @@ func TestSource_WatcherFailsCreatingInformer(t *testing.T) {
 	addCrdEvents(wcrd, r.All())
 
 	// Now start properly and get events
-	cl := fake.NewSimpleDynamicClient(k8sRuntime.NewScheme())
+	gvrToListKind := map[schema.GroupVersionResource]string{
+		{Group: "testdata.istio.io", Version: "v1alpha1", Resource: "Kind1s"}: "Kind1List",
+	}
+	cl := fake.NewSimpleDynamicClientWithCustomListKinds(k8sRuntime.NewScheme(), gvrToListKind)
```

Comment:
Is this necessary

Reply:
Yes they made this required. There might be a better way to do this but since we are planning to delete this code in 1.9 I didn't want to invest too much time dealing with the mess